.devcontainer/db/Dockerfile

```
@@ -5,11 +5,15 @@ LABEL author="OpenOSP"
 LABEL version=0.0.2
 LABEL description="OpenOSP-EMR Docker Image for Development Environment"
 
+# Pre-create root-owned data directory explicitly so mariadb runtime user can use it
+RUN mkdir -p /var/lib/OscarDocument && chown -R mysql:mysql /var/lib/OscarDocument
+
 # Install required packages
-RUN apt-get update && apt-get install -y dos2unix  \
+RUN apt-get update && apt-get install -y dos2unix \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)